packages/components/buttons/accessible-button/src/accessible-button.tsx

```diff
@@ -142,7 +142,7 @@ const AccessibleButton = forwardRef<HTMLButtonElement, TAccessibleButtonProps>(
       buttonProps = {
         type: props.type,
       };
-    } else if (!props.isDisabled) {
```

Comment:
I think the `onKeyPress` should still not be included when the button is disabled.

Reply:
Perhaps we can keep it simple since [the callback](https://github.com/commercetools/ui-kit/blob/main/packages/components/buttons/accessible-button/src/accessible-button.tsx#L128) already take into account the `disabled` state.

Reply:
Yes, I was going to state that but also, it 's a tiny change. so I can just avoid getting to the function as well. 
Updated here: https://github.com/commercetools/ui-kit/pull/2262/commits/79d797993da297e414bdce4149da03670f318093
Thank you

Reply:
Sure. I prefer the other way since it seems cleaner and more maintainable to me, but no big deal.

Reply:
Alright, fine by me. Thanks for double checking.